specs/MultiProfile.md

```diff
@@ -328,6 +362,18 @@ interface ICoreWebView2Profile2 : ICoreWebView2Profile {
   /// See ICoreWebView2CookieManager.
   [propget] HRESULT CookieManager([out, retval] ICoreWebView2CookieManager** cookieManager);
 }
+
+[uuid(1c1ae2cc-d5c2-ffe3-d3e7-7857035d23b7), object, pointer_default(unique)]
+interface ICoreWebView2Profile3 : ICoreWebView2Profile2 {
+  /// All webviews on this profile will be closed, and the profile will be marked for deletion.
+  /// After the Delete() call completes, The render process of webviews on this profile will
+  /// asynchronously exit with the reason:`COREWEBVIEW2_PROCESS_FAILED_REASON_PROFILE_DELETED`.
+  /// See 'COREWEBVIEW2_PROCESS_FAILED_REASON::COREWEBVIEW2_PROCESS_FAILED_REASON_PROFILE_DELETED'
+  /// for more details. The profile directory on disk will be actually deleted when the browser
+  /// process exits. Profile creation will fail if you create a new profile with the same name
+  /// as a profile that is being deleted.
+  HRESULT Delete(); 
```

Comment:
What happens if you delete the 'default' profile? Does that work the same as usual (except that it can be implicitly recreated by creating a new webview2 but without specifying a profile)?

Reply:
The 'default' profile do as non-default profile, the only difference is that you don't need to specify a name when creating it.